docs/installation.rst

```diff
@@ -4,7 +4,7 @@
 Installation
 ============
 
-Tszip can be installed using the standard `pip <https://pypi.org/project/pip/>`_ distribution method::
+Pyslim can be installed using the standard `pip <https://pypi.org/project/pip/>`_ distribution method::
```

Comment:
nice catch here, lol.

Reply:
At least now that fact that it's nicked from tszip is a bit less obvious

> But plagiarize, plagiarize, plagiarize
> Only be sure always to call it please 'research'

https://www.youtube.com/watch?v=gXlfXirQF3A

:)